client/src/app/components/my-tasks/my-tasks.component.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import {BrowserModule} from '@angular/platform-browser';
```

Comment:
Spacing with these import statements is inconsistent - choose import {x} or import { x } - then make all the same please :)